plain-ai.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Plain AI'
+---
+
+Quickly categorize and label new threads without lifting a finger. To turn on Plain’s AI features, go to **Settings → Workflow.\***
+
+<Frame>![Plain AI](/public/images/plain-ai.png)</Frame>
+
+**Auto-labelling**
+
+Plain AI will add 1-2 labels automatically to a thread based on its content. To best leverage Plain AI, make sure you’ve already set up [Labels](/labels) (**Settings → Labels**).
```

Comment:
Do we want to mention that labels won't be added to threads that already have labels (e.g. when a thread is created via the API with a label)

Reply:
Yeah, that's probably a good idea. Do you have some good copy for that?

Reply:
```suggestion
When a thread is created, Plain AI will add 1-2 labels automatically to a thread based on its content. If a thread was created with existing labels, no labels will be added. To best leverage Plain AI, make sure you’ve already set up [Labels](/labels) (**Settings → Labels**).
```